documentation/Windows Event Log (EVT) format.asciidoc

```diff
@@ -168,7 +170,8 @@ The event record (EVENTLOGRECORD) is variable of size and consist of:
 | Offset | Size | Value | Description
 | 0 | 4 | | Size +
 Including the size value
-| 4 | 4 | "LfLe" | Signature (ELF_LOG_SIGNATURE)
+| 4 | 4 | "LfLe" | Signature (ELF_LOG_SIGNATURE) +
+0x654c664c
```

Comment:
why add 0x654c664c?

Reply:
When looking at the file in a hex editor, it's nice to know the numeric values rather than having to calculate them yourself. Especially if you mess up the little endian bit - the value is ```(DWORD)'eLfL'```, which may not be entirely clear to readers seeing "LfLe".

Reply:
Then please add it after the "LfLe" as:

Or `"LfLe" (0x654c664c)`

Then again I've also seen people be confused by hex notation.